crossterm_input/src/input/input.rs

```diff
@@ -44,11 +44,11 @@ impl TerminalInput {
     ///
     /// # Example
     /// ```rust
-    /// let input = input();
-    ///  match input.read_line() {
+    /// let input = crossterm_input::input();
```

Comment:
So this was another concern of me, most of the time people will be using `crossterm_input` through crossterm. All types and functions are re-exported. In that case, this namespace is not right and people might get confused because they don't have access to that namespace.

Reply:
These reexports ... What do you propose? Remove these two examples? Move them to the book / root crate?

Reply:
What do you think [about this](https://github.com/TimonPost/crossterm/pull/207/commits/945e1ba7d13c0cbf7202bf0531f0feee7183bd90)? Tests are passing, these two examples marked as ignored (even in the tests output), ...

Reply:
Ignoring examples might cause old invalid code in the feature. But I much more like it to ignore the compilation then having wrong imports. So that looks good to me

Reply:
At least it's visible in the test output and can be solved in a better way later. Marking as resolved.